it/uk/gov/hmrc/soletraderidentificationfrontend/views/CaptureSautrViewTests.scala

```diff
@@ -96,7 +100,7 @@ trait CaptureSautrViewTests {
     lazy val doc: Document = Jsoup.parse(result.body)
 
     "have the correct title" in {
-      doc.title mustBe Base.Error.error + messages.title
+      doc.title mustBe Base.Error.error + s"${messages.title} - $testDefaultServiceName - GOV.UK"
```

Comment:
Refaque do you think this make more sense ... 
we are using message.title .... but it is only part of the title. Maybe we can modify the messages entries in the way it is indicated.
I know that it is Friday but I have to do these considerations ;-)

<img width="1242" alt="Screenshot 2022-04-22 at 16 31 21" src="https://user-images.githubusercontent.com/4958044/164747225-7a01609c-da0f-4e8c-abd2-340dc2216a0e.png">

<img width="483" alt="Screenshot 2022-04-22 at 16 33 19" src="https://user-images.githubusercontent.com/4958044/164746934-0caae5b5-a185-43ea-b829-7f5125cf8247.png">
fa8.png">